### DISCUSSION TITLE
RPC methods not responding: eth_call, debug_*, trace_*

### DISCUSSION THREAD

**Discussion by Mummyhap** [paradigmxyz/reth#3650] [General]
Hey, this is BlockPI network. As a RPC service provider, we want to run differernt node clients in our network to make it robust. When we are testing RETH, we found that eth_call, debug_*, trace_* methods are not responding. I understand that debug_*, trace_* are still in developing.  But it seems like no one mentioned any issue with eth_call. Just want to make sure you are aware of this. Here is the payload of the RPC,
{
    "method": "eth_call",
    "params": [
        {
            "from": null,
            "to": "0x6b175474e89094c44da98b954eedeac495271d0f",
            "data": "0x70a082310000000000000000000000006E0d01A76C3Cf4288372a29124A26D4353EE51BE"
        },
        "latest"
    ],
    "id": 1,
    "jsonrpc": "2.0"
}
We are very hopeful to see the progress of Reth and look forward to seeing it become fully available as soon as possible.

**Comment by mattsse** [comment]
hey thanks for flagging

this sounds more like an issue so I moved it here https://github.com/paradigmxyz/reth/issues/3655

closing
